build.cake

```diff
@@ -313,8 +313,13 @@ Task("Sign-Binaries")
     if(string.IsNullOrWhiteSpace(secret)) {
         throw new InvalidOperationException("Could not resolve signing secret.");
     }
+    // Get the user.
+    var user = EnvironmentVariable("SIGNING_USER");
+    if(string.IsNullOrWhiteSpace(user)) {
+        throw new InvalidOperationException("Could not resolve signing user.");
+    }
 
-    var client = File("./tools/SignClient/tools/SignClient.dll");
+    var client = File("./tools/SignClient/tools/netcoreapp2.0/SignClient.dll");
```

Comment:
Currently we're building using the 1.0.7 SDK(2.0 is on the way just not yet), so I suspect this version requires .NET Core 2.0? So we should use the netcoreapp1.1 moniker instead right?

Reply:
k

Reply:
Excellent 👍  , I'll take this for a spin.